src/subsplit_dag.cpp

```diff
@@ -38,6 +38,27 @@ SubsplitDAG::SubsplitDAG(size_t taxon_count,
          "Taxon ids do not cover all indices from 0 to taxon_count.");
 }
 
+SubsplitDAG::SubsplitDAG(SubsplitDAG &prepend, PrependTag)
+    : storage_{prepend.storage_, PrependTag{}},
```

Comment:
If you want to make a move to start using brace initialization, say so, but this isn't our current guideline setup.

Reply:
Sorry, will change that.